packages/driver/src/cypress/error_messages.js

```diff
@@ -1328,6 +1328,7 @@ module.exports = {
 
   should: {
     chainer_not_found: 'The chainer `{{chainer}}` was not found. Could not build assertion.',
+    language_chainer: 'The chainer `{{chainer}}` contains only one language chainer. Please use a valid assertion.',
```

Comment:
This error is a tough one to write.... 

I would also add a `docsUrl` property to this error that links to `https://on.cypress.io/assertions`. So this will need to be split into `message` / `docsUrl` like seen in some other errors. 

```suggestion
    language_chainer: 'The chainer `{{chainer}}` is a language chainer provided to improve the readability of your assertions, not an actual assertion. Please provide a valid assertion.',
```

Reply:
Thanks for the error message suggestion. The `docsUrl` is added to the error content now :)